lambdanotes/src/components/Note.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import './Note.css';
+import { Link } from 'react-router-dom';
+
+export default class Note extends Component {
+    constructor(props) {
+    super(props);
```

Comment:
```js

export default class Note extends Component {
    constructor(props) {
        super(props);
        this.state = {
            note: {},
            modal: false
        };
    }

```

what you have works,  but newlines and indentations make the code a bit more readable. Personal preference, really, so do whatever your manager tells you when you get to the workplace :)